<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>searchPlusDockWidget</class>
 <widget class="QDialog" name="searchPlusDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search plus</string>
  </property>
  <widget class="QTabWidget" name="tab_main">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>331</width>
     <height>116</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_network">
    <attribute name="title">
     <string>Network</string>
    </attribute>
    <widget class="QComboBox" name="network_code">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="network_geom_type">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_type">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_code">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>45</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Code:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_hydro">
    <attribute name="title">
     <string>Hydro</string>
    </attribute>
    <widget class="QComboBox" name="expl_name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_connec_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Exploitation:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_hydrometer_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>45</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Hydrometer:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="hydro_id">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_address">
    <attribute name="title">
     <string>Address</string>
    </attribute>
    <widget class="QComboBox" name="address_street">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="address_number">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="address_postal_code">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="address_exploitation">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_municipality">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Municipality:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_postcode">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>5</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Post code:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_postnumber">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>45</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Number:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_street">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>45</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Street:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>WorkCat</string>
    </attribute>
    <widget class="QComboBox" name="workcat_id">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
